pkg/rc/replication_controller_test.go

```diff
@@ -832,6 +832,16 @@ func TestNoOpIfNodeTransferInProgress(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	err = applicator.SetLabel(labels.NODE, "old.123", "nodeQuality", "good")
+	Assert(t).IsNil(err, "expected no error labeling old.123")
+
+	rcFields.ReplicasDesired = 1
+
+	err = rc.meetDesires(rcFields)
+	if err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
What is the purpose of the added code in this test file?

Reply:
This labels the old node in a staged node transfer as belonging to the RC. Without this, the RC would see that it had an in progress node transfer but then see that it was not on the "old node", so it would try to roll back the transfer.